packages/utilities/crypto/lib/types.web.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Represents a public cryptographic key
+ */
+export interface PublicKey {
+	/** Returns the public key bytes */
+	bytes(): Promise<Uint8Array>;
```

Comment:
naming-things nitpick: "bytes" is not a serialisation format - ideally the method name would tell me a format

Reply:
heh yeah, I wonder if this would be a good time to change that method name

Reply:
I revamped the api in 9d6673bbf09fbf17593375a9cdcef7f10d9b8ee5, let me know what you think